src/net_processing.cpp

```diff
@@ -491,47 +491,47 @@ struct CNodeState {
     //! The peer's address
     const CService address;
     //! The best known block we know this peer has announced.
-    const CBlockIndex *pindexBestKnownBlock;
+    const CBlockIndex* pindexBestKnownBlock{nullptr};
     //! The hash of the last unknown block this peer has announced.
-    uint256 hashLastUnknownBlock;
+    uint256 hashLastUnknownBlock{};
```

Comment:
why not 
```suggestion
    uint256 hashLastUnknownBlock{0};
```
?

Reply:
or use `uint256::ZERO` ?

Reply:
I think all three variants are the same, it shouldn't matter which one is picked